Week0/ysh2002m/mission1/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>반응형 네비게이션 바</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <nav class="navbar">
+        <div class="logo">
+            <a href="#"><img src="UMC_logo.png" alt="UMC 로고"></a>
+        </div>
+        <ul class="nav-links">
+            <li><a href="#">소개</a></li>
+            <li><a href="#">프로젝트</a></li>
+            <li><a href="#">Q&A</a></li>
+            <li><a href="#" class="apply">지원하기</a></li>
```

Comment:
이 부분에만 class 이름을 사용한 스타일링이나 추가적 기능이 없다면, `class="apply"` 부분을 제거하는 것이 더 깔끔한 코드가 될 것 같습니다!

```suggestion
            <li><a href="#">지원하기</a></li>
```